csrankings-c.csv

```diff
@@ -503,6 +503,7 @@ Chenshu Wu,University of Hong Kong,https://cswu.me,q9llreMAAAAJ
 Chentao Wu,Shanghai Jiao Tong University,http://www.cs.sjtu.edu.cn/~wuct,RAs-wnEAAAAJ
 Chenxi Zhang,Tongji University,https://sse.tongji.edu.cn/Data/View/2825,NOSCHOLARPAGE
 Chenyang Lu 0001,Washington University in St. Louis,http://www.cs.wustl.edu/~lu,tCq7Wx0AAAAJ
+Chenye Wu,CUHK-SZ,www.wuchenye.cn,Eivoa5IAAAAJ

```

Comment:
Home page ill-formed. Please clarify relationship between SSE and SDS at CUHK. It's not clear that faculty in SSE can solely supervise PhDs in _Computer Science_.

Reply:
We appreciate very much your responses to our previous request regarding listing CUHK-Shenzhen in CSRankings.
We have followed your suggestions and amended the information to be complete and accurate. Next is a summary of our clarifications and amendments.
The Chinese University of Hong Kong, Shenzhen(CUHK-Shenzhen) has two computer science and engineering (CSE) related programs (and faculty), under the School of Data Science(SDS) and School of Science and Engineering(SSE). Details can be found at the following links.
Homepage: SDS https://sds.cuhk.edu.cn/en ; SSE https://sse.cuhk.edu.cn/en
PhD program in CS area: SDS https://sds.cuhk.edu.cn/en/phd-programmes ; SSE https://sse.cuhk.edu.cn/en/taxonomy/term/157
Academic Area: SDS https://sds.cuhk.edu.cn/en/page/525 ; SSE https://sse.cuhk.edu.cn/en/page/30